codegen/src/main/java/software/amazon/awssdk/codegen/IntermediateModelBuilder.java

```diff
@@ -157,9 +161,28 @@ public IntermediateModel build() {
 
         setSimpleMethods(trimmedModel);
 
+        validateOperations(trimmedModel);
+
         return trimmedModel;
     }
 
+    /**
+     * Validates that operations that use GET, DELETE or HEAD do not have an HTTP body.
+     */
+    private void validateOperations(IntermediateModel model) {
+        List<String> methods = model.getOperations()
+                                    .entrySet()
```

Comment:
values().stream()? You're not using the keys below.

Reply:
Updated